index.rst

```diff
@@ -0,0 +1,304 @@
+Mappit API
+------------
+
+Australian address validation and reverse geocoding made easy.
+
+
+Authentication
+--------------
+
+After registering for an account, login to the `Dashboard`_ and generate an API key.
+You can then use the generated API key in the ``X-Api-Key`` HTTP header on your `GraphQL`_ requests.
+
+.. sourcecode:: http
+
+  POST /graphql HTTP/2
+  Host: api.mappit.app
+  X-Api-Key: <your-api-key>
+  Content-Type: application/json
+
+Usage
+-----
+
+Search address
+~~~~~~~~~~~~~~
+
+You can search for an address using this query. This is built as a search-as-you-type query, and is also typo tolerant.
+
+.. graphiql::
+  :query:
+    {
+      searchAddress(query: "500 col"){
+        gnafId
+        displayAddress
+        location{
+          lat
+          lon
+        }
+        components{
+          buildingName
+          number
+          street
+          suburb
+          state
+          postcode
+        }
+        score
+      }
+    }
+  :response:
+    {
+      "data": {
+        "searchAddress": [
+          {
+            "gnafId": "GAVIC424236450",
+            "displayAddress": "500 Collins Street, Melbourne VIC 3000",
+            "location": {
+              "lat": -37.81784378,
+              "lon": 144.95761462
+            },
+            "components": {
+              "buildingName": "",
+              "number": "500",
+              "street": "Collins Street",
+              "suburb": "Melbourne",
+              "state": "VIC",
+              "postcode": 3000
+            },
+            "score": 1
+          }
+        ]
+      }
+    }
+
+You can also optionally filter the search results to a specific state by providing a value for the ``state`` parameter.
+
+.. graphiql::
+  :query:
+    {
+      searchAddress(query: "500 col", state: NSW){
+        gnafId
+        displayAddress
+        location{
+          lat
+          lon
+        }
+        components{
+          buildingName
+          number
+          street
+          suburb
+          state
+          postcode
+        }
+        score
+      }
+    }
+  :response:
+    {
+      "data": {
+        "searchAddress": [
+          {
+            "gnafId": "GANSW716730214",
+            "displayAddress": "500 Colemans Lane, Milvale NSW 2594",
+            "location": {
+              "lat": -34.25705103,
+              "lon": 147.86797566
+            },
+            "components": {
+              "buildingName": "",
+              "number": "500",
+              "street": "Colemans Lane",
+              "suburb": "Milvale",
+              "state": "NSW",
+              "postcode": 2594
+            },
+            "score": 1
+          }
+        ]
+      }
+    }
+
+The ``state`` parameter supports the following values: ``TAS, VIC, NSW, ACT, QLD, SA, NT, WA, OT``
+
+Retrieve property
+~~~~~~~~~~~~~~~~~
+
+If you already know the gnaf ID of a property, you can use it to retrieve the details of that property.
```

Comment:
```suggestion
If you already know the G-NAF ID of a property, you can use it to retrieve the details of that property.
```

Might as well format it properly for the docs :/

Reply:
Yep makes sense.